sdk/workflow-tengo/src/exec/exec.tpl.tengo

```diff
@@ -204,7 +229,11 @@ self.body(func(inputs) {
 		"assets,?": { any: validation.resource() },
 		"assetDescriptors,?": { any: {asset: desc.assetScheme } },
 		"assetRules,?": { any: { any: ["string"] } },
-		"assetCache,?": { any: "number" }
+		"assetCache,?": { any: "number" },
+		"quota,?": {
```

Comment:
Could we move it into runOptions? So then no additional value resources will be added to this already big template. `runOptions` is basically 1 JSON value resource

Reply:
Moved it away into a separate service input. It will not affect deduplication at all, because the resources availability is not expected to affect the final execution result, if it succeeds. If it fails, it would be re-run anyway.